truffle/src/main/java/org/jruby/truffle/core/exception/NameErrorLayout.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2015, 2016 Oracle and/or its affiliates. All rights reserved. This
```

Comment:
If you didn't copy this code from somewhere else, it should probably just be 2016.